app/scripts/lib/fxa-client.js

```diff
@@ -399,25 +403,31 @@ define(function (require, exports, module) {
         clientOptions.resume = options.resume;
       }
 
+      setMetricsContext(clientOptions, options);
+
       return client.passwordForgotResendCode(
         email,
         passwordForgotToken,
         clientOptions
       );
     }),
 
-    completePasswordReset: withClient((client, originalEmail, newPassword, token, code, relier) => {
+    completePasswordReset: withClient((client, originalEmail, newPassword, token, code, relier, options = {}) => {
       const email = trim(originalEmail);
 
-      return client.passwordForgotVerifyCode(code, token)
+      var clientOptions = {
+        keys: wantsKeys(relier),
+        sessionToken: true
+      };
+
+      setMetricsContext(clientOptions, options);
+
+      return client.passwordForgotVerifyCode(code, token, clientOptions)
```

Comment:
I think should be ok. The `passwordForgotVerifyCode` was updated to accept the [extra options](https://github.com/mozilla/fxa-js-client/blob/master/client/FxAccountClient.js#L546) and auth-server [endpoint](https://github.com/mozilla/fxa-auth-server/blob/master/lib/routes/password.js#L480) was updated last train to accept them too. @rfk had the foresight to get it in before the train cut so that we could avoid having to wait :)